packages/protocol/test-sol/unit/common/CeloDistributionSchedule.t.sol

```diff
@@ -136,15 +131,15 @@ contract CeloDistributionScheduleTest_initialize is CeloDistributionScheduleTest
     celoToken.setCeloTokenDistributionScheduleAddress(address(celoDistributionSchedule));
 
     vm.prank(celoDistributionOwner);
-    celoDistributionSchedule.initialize(registryAddress);
+    celoDistributionSchedule.initialize(REGISTRY_ADDRESS);
   }
 
   function test_ShouldSetAOwnerToCeloDistributionScheduleInstance() public {
     assertEq(celoDistributionSchedule.owner(), celoDistributionOwner);
   }
 
   function test_ShouldSetRegistryAddressToCeloDistributionScheduleInstance() public {
-    assertEq(address(celoDistributionSchedule.registry()), proxyAdminAddress);
```

Comment:
I replaced this with `REGISTRY_ADDRESS`:

```diff
- assertEq(address(celoDistributionSchedule.registry()), proxyAdminAddress);
+ assertEq(address(celoDistributionSchedule.registry()), REGISTRY_ADDRESS);
```

Check if that captures the essence of the test. I have a hunch this might be related to this [comment](https://github.com/celo-org/celo-monorepo/pull/11143#discussion_r1679374208) on the use of `registryAddress = proxyAdminAddress`

Reply:
As far as I can see, my change is correct. The unit test passes as expected. 

To double-check, when I replace this with `PROXY_ADMIN_ADDRESS`, the test fails. 

```diff
- assertEq(address(celoDistributionSchedule.registry()), proxyAdminAddress);
+ assertEq(address(celoDistributionSchedule.registry()), PROXY_ADMIN_ADDRESS);
```

I'm happy with the change to `REGISTRY_ADDRESS` because it's less confusing that `proxyAdminAddress = registryAddress` further up.

```diff
- assertEq(address(celoDistributionSchedule.registry()), proxyAdminAddress);
+ assertEq(address(celoDistributionSchedule.registry()), REGISTRY_ADDRESS);
```